### ISSUE TITLE
[GitHub] The "My Issues" Command displays issues on FOSS projects that I commented on but unsubscribed

### ISSUE THREAD

**Issue by yesseruser** [raycast/extensions#15941]
### Extension

https://www.raycast.com/raycast/github

### Raycast Version

1.88.4

### macOS Version

15.2

### Description

The My Issues command lists issues that I'm no longer interested in. I would like the extension to ignore issues I unsubscribed from or give me the option to hide certain issues

### Steps To Reproduce

1. Make sure you commented on an open issue you unsubscribed from.
2. Open Raycast
3. Run the "My Issues" command
4. Observe the issue

**Comment by thomaslombart** [comment]
Thanks for reporting this bug. Would you like to work on it? If yes, here's our [guide on extension contributions](https://developers.raycast.com/basics/contribute-to-an-extension) so that you can open up a pull request. Feel free to contact me here or in the [Slack community](https://raycast.com/community) if you need further help.

**Comment by yesseruser** [comment]
> Thanks for reporting this bug. Would you like to work on it? If yes, here's our [guide on extension contributions](https://developers.raycast.com/basics/contribute-to-an-extension) so that you can open up a pull request. Feel free to contact me here or in the [Slack community](https://raycast.com/community) if you need further help.

I could try, but if somebody else wants to work on this, I'm not stopping them.

**Comment by antonengelhardt** [comment]
@yesseruser I can check on myself next week... do you know about the filters that the my issues command has in the settings? Also, the extension is pretty limited in what it can do, as it uses octokit (GitHub SDK). If you want to experiment a bit in the meantime, go wild

**Comment by yesseruser** [comment]
> @yesseruser I can check on myself next week... do you know about the filters that the my issues command has in the settings? Also, the extension is pretty limited in what it can do, as it uses octokit (GitHub SDK). If you want to experiment a bit in the meantime, go wild 

I'm not sure I know about those filters, but feel free to check the issue yourself.

**Comment by yesseruser** [comment]
> [@yesseruser](https://github.com/yesseruser) I can check on myself next week... do you know about the filters that the my issues command has in the settings? Also, the extension is pretty limited in what it can do, as it uses octokit (GitHub SDK). If you want to experiment a bit in the meantime, go wild

@thomaslombart you're right, I didn't know those filters existed. Thank you for letting me know.
